src/aten/EmptyTensor.cpp

```diff
@@ -1,6 +1,7 @@
 #define TORCH_ASSERT_NO_OPERATORS
 #include <ATen/Context.h>
 #include <ATen/EmptyTensor.h>
+#include <ATen/xpu/EmptyTensor.h>
```

Comment:
Since we have included the xpu/EmptyTensor, is ATen/EmptyTensor still required?

Reply:
`ATen/xpu/EmptyTensor.h` is the exported interface of `EmptyTensor.cpp`
and `ATen/EmptyTensor.h` is for `at::detail::empty_generic` which is used in this  `EmptyTensor.cpp` file.

Reply:
Then my question will be why `ATen/xpu/EmptyTensor.h` includes `ATen/EmptyTensor.h`?

Reply:
Sorry, but `ATen/xpu/EmptyTensor.h` does not include `ATen/EmptyTensor.h`.